### DISCUSSION TITLE
4.2.3 docker image

### DISCUSSION THREAD

**Discussion by coltonfreeman26** [eclipse-rdf4j/rdf4j#4494] [General]
Hello all,
Do you know if there is a date when 4.2.3 will be available on docker hub?

**Comment by barthanssens** [comment]
Hello @coltonfreeman26 , sorry about the delay, the  4.2.3 image should now be available on hub.docker.com

**Reply by coltonfreeman26** [reply]
Not to worry. Thank you for the response, our automation tool just picked it up.
